source/opt/decoration_manager.cpp

```diff
@@ -46,6 +46,21 @@ void DecorationManager::RemoveDecorationsFrom(
   for (ir::Instruction* inst : decorations_info.indirect_decorations) {
     assert(inst->opcode() == SpvOpGroupDecorate ||
            inst->opcode() == SpvOpGroupMemberDecorate);
+
+    std::vector<ir::Instruction*> group_decorations_to_keep;
+    const uint32_t group_id = inst->GetSingleWordInOperand(0u);
+    const auto group_iter = id_to_decoration_insts_.find(group_id);
+    for (ir::Instruction* group_decoration :
+         group_iter->second.direct_decorations)
```

Comment:
Please add the braces.  It is easier to see what is in the scope of the for loop.

Reply:
Will do